roles/tomcat/molecule/default/molecule.yml

```diff
@@ -1,75 +1,32 @@
 ---
-scenario:
-  test_sequence:
-    - lint
-    - cleanup
-    - destroy
-    - syntax
-    - create
-    - prepare
-    - converge
-    - idempotence
-    - side_effect
-    - verify
-    - cleanup
-    - destroy
 dependency:
   name: galaxy
 driver:
   name: docker
 platforms:
-  - name: instance
-    image: centos:7
+  - name: tomcat
+    image: ${MOLECULE_ROLE_IMAGE:-centos:7}
+    dockerfile: ../../../../tests/molecule/Dockerfile.j2
+    command: "/lib/systemd/systemd"
     privileged: true
     tmpfs:
       - /run
       - /run/lock
       - /tmp
     volume_mounts:
       - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
-    command: /sbin/init
-    exposed_ports:
-      - 80/tcp
-      - 443/tcp
-      - 5432/tcp
-      - 8161/tcp
-      - 8080/tcp
-      - 8082/tcp
-      - 8083/tcp
-      - 8090/tcp
-      - 8099/tcp
-      - 8983/tcp
-      - 9090/tcp
-    published_ports:
-      - 0.0.0.0:80:80/tcp
-      - 0.0.0.0:443:443/tcp
-      - 0.0.0.0:5432:5432/tcp
-      - 0.0.0.0:8161:8161/tcp
-      - 0.0.0.0:8080:8080/tcp
-      - 0.0.0.0:8082:8082/tcp
-      - 0.0.0.0:8083:8083/tcp
-      - 0.0.0.0:8090:8090/tcp
-      - 0.0.0.0:8099:8099/tcp
-      - 0.0.0.0:8983:8983/tcp
-      - 0.0.0.0:9090:9090/tcp
 provisioner:
   name: ansible
   ansible_args:
     - --inventory=../../inventory_local.yml
-    - --limit=instance
+    - --limit=tomcat
   inventory:
     links: 
       group_vars: ../../../../group_vars
-  playbooks:
-    converge: converge.yml
 verifier:
   name: testinfra
-  directory: tests
   env:
     TEST_HOST: "localhost"
-    ANSIBLE_TIMEOUT: "30"
-    ANSIBLE_SSH_RETRIES: "15"
-    PYTHONWARNINGS: "ignore:.*U.*mode is deprecated:DeprecationWarning"
   options:
-    v: 1
+    verbose: true
     s: true
```

Comment:
Did you intend to keep that verifier switch in the tomcat tests while it's been removed from all others?

Reply:
no it's only a leftover, I didn't have found a good reason to explicitly disable i/o capture and I get rid of them

verbose is useful because tests looks stuck until the end and was confusing me :D